src/styles/blocks/display.scss

```diff
@@ -0,0 +1,42 @@
+.main__display {
+  position: relative;
+}
+
+.display {
+  @extend %alignment;
+  @include circle($displaySize);
+  border: 2px solid #000;
+
+  &__hands-center {
+    position: absolute;
+    @include circle($handsSize);
+    background-color: $colorBlue;
+  }
+
+  &__second-hand {
+    @extend %postionHands;
+    @extend %handsview;
+    height: 220px;
+    background-color: $colorBlue;
+    transform-origin: bottom center;
+    animation: animation 60s linear infinite;
+  }
+
+  &__minute-hand {
+    @extend %postionHands;
+    @extend %handsview;
+    height: 150px;
+    background-color: #f00;
+    transform-origin: bottom center;
+    animation: animation 3600s linear infinite;
+  }
+
+  @keyframes animation {
+    from {
+      transform: rotate(0deg);
+    }
+    to {
+      transform: rotate(360deg);
+    }
+  }
```

Comment:
would be better to move keyframe to separate file

Reply:
I thought that you mean to make it part of the block. changed

Reply:
- please check shift of your arrows

I was able to do this only with the help of additional positioning. If this is not correct -  please, help me.